test/e2e/Tests/Helpers/DurableHelpers.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Text.Json.Nodes;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+internal class DurableHelpers
+{
+    internal static string ParseStatusQueryGetUri(HttpResponseMessage invocationStartResponse)
+    {
+        string? responseString = invocationStartResponse.Content?.ReadAsStringAsync().Result;
+        if (string.IsNullOrEmpty(responseString))
+        {
+            return string.Empty;
+        }
+        JsonNode? responseJsonNode = JsonNode.Parse(responseString);
+        if (responseJsonNode == null)
+        {
+            return string.Empty;
+        }
+
+        string? statusQueryGetUri = responseJsonNode["StatusQueryGetUri"]?.GetValue<string>();
+        return statusQueryGetUri ?? string.Empty;
+    }
+    internal static string GetRuntimeStatus(string statusQueryGetUri)
+    {
+        HttpClient client = new HttpClient();
```

Comment:
We should avoid creating `HttpClient` objects and letting them get garbage collected since that can create a resource leak (customers run into this problem a lot). A better practice is to create a shared `static readonly HttpClient` that can be reused. They are safe for use by multiple threads concurrently, so you don't need to worry about synchronizing access to it.

Reply:
Done